frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts

```diff
@@ -116,10 +115,15 @@ export class NewMeetingComponent extends BaseComponent implements OnInit, OnDest
         this.patchFormValues();
         this.setValidation();
 
-        this.teamService.currentTeamEmitted$.subscribe((teamId) => {
+        this.teamService.currentTeamEmitted$.pipe(this.untilThis).subscribe((teamId) => {
             this.currentTeamId = teamId;
-            this.getTeamMembersOfCurrentUser(teamId);
+            this.subscribeToSearchTeamMembers();
+
+            this.userService.userChangedEvent$.pipe(this.untilThis).subscribe((resp) => {
+                this.addCurrentTeamMemberToList(resp?.id);
+            });
```

Comment:
[Stop! You've violated the law!](https://www.youtube.com/watch?v=My0lzMuNcHI)

Reply:
the code has been changed in accordance with current legislation )

Reply:
There are still nested subscription

Reply:
fixed